packages/react-component-library/src/components/Alert/Alert.stories.tsx

```diff
@@ -33,3 +36,23 @@ WithoutTitle.args = {
+  background-size: contain;
+  height: 3rem;
+`
+
+ArbitraryContent.args = {
+  title: 'Using arbitrary JSX as content',
+  children: <div><StyledLogoContainer />
```

Comment:
Although it's now loosened and possible to do this i'd rather us include a more simplistic example.

Maybe just `<strong>Arbitrary JSX</strong>` or something?

My thinking around it is I don't want people to think it's ok to put images etc in an Alert.